modules/distribution-service-acl/src/main/java/org/opencastproject/distribution/acl/AclDistributionService.java

```diff
@@ -1,280 +0,0 @@
-      throw new IllegalStateException("Distribution directory must be set (org.opencastproject.download.directory)");
-    this.distributionDirectory = new File(ccDistributionDirectory);
-    logger.info("Download distribution directory is {}", distributionDirectory);
-    this.distributionChannel = OsgiUtil.getComponentContextProperty(cc, CONFIG_KEY_STORE_TYPE);
-  }
-
-  public String getDistributionType() {
-    return this.distributionChannel;
-  }
-
-  @Override
-  public Job distribute(String channelId, MediaPackage mediapackage, String elementId)
-          throws DistributionException, MediaPackageException {
-    notNull(mediapackage, "mediapackage");
-    notNull(elementId, "elementId");
-    notNull(channelId, "channelId");
-
-    try {
-      return serviceRegistry.createJob(JOB_TYPE, Operation.Distribute.toString(),
-                                       Arrays.asList(channelId,
-                                                     MediaPackageParser.getAsXml(mediapackage),
-                                                     elementId), distributeJobLoad);
-    } catch (ServiceRegistryException e) {
-      throw new DistributionException("Unable to create a job", e);
-    }
-  }
-
-  protected MediaPackageElement distributeElement(String channelId, MediaPackage mediaPackage, String elementId)
-          throws DistributionException, MediaPackageException {
-    try {
-      File source = getAclXmlAttachmentFile(mediaPackage);
-      if (source == null) {
-        throw new DistributionException("No available acl for mediapackage " + mediaPackage);
-      }
-
-      File destination = new File(PathSupport.concat(new String[] { distributionDirectory.getAbsolutePath(),
-              mediaPackage.getIdentifier().compact(), elementId, elementId + ".acl" }));
-
-      // Create the parent directory if it doesn't exist.
-      try {
-        FileUtils.forceMkdir(destination.getParentFile());
-      } catch (IOException e) {
-        throw new DistributionException("Unable to create " + destination.getParentFile(), e);
-      }
-      logger.info("Distributing {} to {}", elementId, destination);
-      // Copy the acl from source to destination.
-      try {
-        FileSupport.copy(source, destination);
-      } catch (IOException e) {
-        throw new DistributionException("Unable to copy " + source + " to " + destination, e);
-      }
-
-      return (MediaPackageElement)getAclXmlAttachment(mediaPackage);
-    } catch (Exception e) {
-      logger.warn("Could not distribute acl XML with " + elementId + " from " + mediaPackage.getIdentifier().compact(), e);
-      return null;
-    }
-  }
-
-  /**
-   * @param mediaPackage The media package to find the attachment that has an ACL.
-   * @return Returns the attachment if possible or null.
-   */
-  private Attachment getAclXmlAttachment(MediaPackage mediaPackage) {
-    Attachment[] aclAttachments = mediaPackage.getAttachments(new MediaPackageElementFlavor("text", "acl"));
```

Comment:
The hard-coded flavor text/acl is very much the indication that all the distribute-acl stuff is obsolete